cfgov/jinja2/v1/external-site/index.html

```diff
@@ -28,18 +28,14 @@
 </script>
 {% endblock javascript %}
 
-{% block content_modifiers -%}
-    external-site_container
-{%- endblock %}
-
 {% block content_main %}
-    <section class='padded-container'>
+    <div class="external-site_container">
```

Comment:
This element can be removed entirely (keeping all of its descendants, of course).

Reply:
It's the container used by the JS. We could refactor that differently, but it's probably a good idea, component-wise, to always have a top-level named container for things.

Reply:
Hmm... right. I don't really think of this kind of thing as a "component" (and not sure I can put my finger on why or defend that perspective), but I suppose it's not worth the refactoring here.

Reply:
I shortened it to just `external-site` for the container.